contracts/BIFI/infra/BeefyFeeConfigurator.sol

```diff
@@ -0,0 +1,143 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+
+interface IStrategy {
+    function vault() external view returns (address);
+}
+
+contract BeefyFeeConfigurator is OwnableUpgradeable {
+
+    struct FeeCategory {
+        uint256 total;
+        uint256 beefy;
+        uint256 call;
+        uint256 strategist;
+        string label;
```

Comment:
What are we using label for? It might be helpful for others to add a comment in here about what each item represents.

Reply:
Label is intended as general descriptor of the fee category, inspired by Aave eModes. We don't have to use it, but could be something to use later on. I've added comments to describe each property